src/index.css

```diff
@@ -1,3 +1,13 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
+body {
+  background-color: #4369b2;
+}
+
+footer {
+  position: fixed;
+  left: 0;
```

Comment:
* Kindly stick to the font family given in the design 
![image](https://github.com/abel-tefera/fbi-most-wanted/assets/52497246/b29e5044-7bfd-416f-a8e5-28ec54f8c566)